modin/pandas/test/dataframe/test_reduce.py

```diff
@@ -444,7 +445,11 @@ def test_value_counts_categorical():
         # The order of HDK categories is different from Pandas
         # and, thus, index comparison fails.
         def comparator(df1, df2):
-            assert_series_equal(df1._to_pandas(), df2, check_index=False)
+            # Perform our own non-strict version of dtypes equality check
+            assert_dtypes_equal(df1, df2)
```

Comment:
`.groupby().count()` result now have `int32` instead of `int64` type, so making a non-strict equality check